webpack.config.js

```diff
@@ -102,20 +108,20 @@ jsLoader = {
 
 if ( CALYPSO_ENV === 'development' ) {
 	webpackConfig.plugins.push( new PragmaCheckPlugin() );
-	webpackConfig.plugins.push( new webpack.HotModuleReplacementPlugin() );
+	//webpackConfig.plugins.push( new webpack.HotModuleReplacementPlugin() );
 	webpackConfig.entry[ 'build-' + CALYPSO_ENV ] = [
 		'webpack-dev-server/client?/',
-		'webpack/hot/only-dev-server',
+		//'webpack/hot/only-dev-server',
 		path.join( __dirname, 'client', 'boot' )
 	];
 
 	// Add react hot loader before babel-loader
-	jsLoader.loaders = [ 'react-hot' ].concat( jsLoader.loaders );
+	//jsLoader.loaders = [ 'react-hot' ].concat( jsLoader.loaders );
 } else {
 	webpackConfig.entry[ 'build-' + CALYPSO_ENV ] = path.join( __dirname, 'client', 'boot' );
 	webpackConfig.devtool = false;
 }
 
-webpackConfig.module.loaders = [ jsLoader ].concat( webpackConfig.module.loaders );
+webpackConfig.module.loaders.push( jsLoader );
```

Comment:
I find this really confusing. Seems like the JS loader should be last, but ... we have it first? Both work. Until you try to use something like HappyPack and then it has to be last. Need to understand how the sections loader and this loader play together.